sql/moz-fx-data-marketing-prod/adjust_derived/firefox_mobile_installs_v1/query.sql

```diff
@@ -0,0 +1,31 @@
+SELECT
+  adjust.date,
+  adjust.network,
+  adjust.campaign,
+  adjust.adgroup,
+  adjust.creative,
+  UPPER(adjust.country) AS country,
+  country.name AS country_name,
+  adjust.os,
+  adjust.device_type,
+  SUM(adjust.clicks) AS clicks,
+  SUM(adjust.installs) AS installs
+FROM
+  `ga-mozilla-org-prod-001.Adjust.deliverable_*` AS adjust
+LEFT JOIN
+  `moz-fx-data-shared-prod.static.country_codes_v1` AS country
+ON
+  UPPER(adjust.country) = country.country
+WHERE
+  adjust.app = 'Firefox Android and iOS'
+  AND adjust.date >= '2022-01-01'
```

Comment:
Better to limit the tables scanned.
```suggestion
  AND _TABLE_SUFFIX >= '20220101'
```